src/main/java/Main.java

```diff
@@ -1,5 +1,113 @@
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+
+import org.apache.commons.io.FileUtils;
+import org.json.JSONArray;
+import org.json.JSONObject;
+
 public class Main {
+
+    public static final String ANSI_RESET = "\u001B[0m";
+    public static final String ANSI_BLUE = "\u001B[34m";
+    public static final String ANSI_MAGENTA = "\u001b[35m";
+
+    public static JSONObject startRoom;
+    public static JSONObject endRoom;
+
+    public static HashMap<String, Room> roomMap = new HashMap<>();
+    public static HashMap<String, Item> itemMap = new HashMap<>();
+
+    public static String playerName;
+
     public static void main (String[] args){
-        System.out.println("Welcome to Software Design");
+
+        System.out.println(ANSI_BLUE + "\n Welcome to the Text Adventure Game. \n Please enter the name of the json file you want to load. \n"
+        + ANSI_RESET);
+        BufferedReader in = new BufferedReader(new InputStreamReader(System.in));
+
+        try {
```

Comment:
Your main is quite long and complex with too many levels of nesting. Try to split up the functionality into smaller, more manageable parts.